media-tv/plex-media-server/plex-media-server-1.13.9.5456.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd unpacker user
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+COMMIT="ecd600442"
+
+_APPNAME="plexmediaserver"
+_USERNAME="plex"
+_SHORTNAME="${_USERNAME}"
+_FULL_VERSION="${PV}-${COMMIT}"
+
+SRC_URI="
+	amd64? ( https://downloads.plex.tv/${PN}/${_FULL_VERSION}/${_APPNAME}_${_FULL_VERSION}_amd64.deb )
+	x86? ( https://downloads.plex.tv/${PN}/${_FULL_VERSION}/${_APPNAME}_${_FULL_VERSION}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist strip"
+IUSE="avahi"
+
+DEPEND="sys-apps/fix-gnustack"
+RDEPEND="avahi? ( net-dns/avahi )"
+
+QA_DESKTOP_FILE="usr/share/applications/plexmediamanager.desktop"
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=( "usr/lib/${_APPNAME}/.*" )
+
+PATCHES=( "${FILESDIR}"/plexmediamanager.desktop.patch )
```

Comment:
This file doesn't exist, did you mean to add it with this PR?

Reply:
No, I did not :-| I've made some more changes looking into previous bugs.